dist/react-number-format.js

```diff
@@ -202,50 +202,8 @@ return /******/ (function(modules) { // webpackBootstrap
 	      }
 	    }
 	  }, {
-	    key: 'formatWithPattern',
```

Comment:
There was many things relate to formatWithPattern which we don't use at the moment. So I removed them.